map/map_class.py

```diff
@@ -54,17 +54,50 @@ def get_lateral_lanes(self, lane_id, road_name):
 
         return lateral_lanes
 
+    def update_lane_info(self, road_id, l_id, dec):
+        lane_id = None
+
+        for road in self.__roads:
```

Comment:
make self.__roads a dict instead of a list, so we don't have to traverse every time